source/content/plugins-recommended.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Recommended WordPress Plugins
+description: A list of WordPress plugins that are recommended for use on Pantheon.
+tags: [plugins]
+contributors: [jspellman814]
+contenttype: [doc]
+innav: [true]
+categories: [issues]
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+reviewed: "2023-05-23"
+---
+
+This page lists WordPress plugins that Pantheon recommends using for various use cases. This is not a comprehensive list, and will continue to evolve over time.
+
+## Index WP MySQL For Speed
+
+<ReviewDate date="2023-05-23" />
+
+[Index WP MySQL For Speed](https://wordpress.org/plugins/index-wp-mysql-for-speed/) makes your MySQL database work more efficiently by adding high-performance keys to the tables you choose.
+
+On a modestly sized site (with a few users and a few hundred posts) your database may be fast enough without these keys. The speed improvements are most noticeable on larger sites with many posts and products.
+
+## WordPress Native PHP Sessions
+
+<ReviewDate date="2023-05-23" />
+
+[WordPress Native PHP Sessions](https://wordpress.org/plugins/wp-native-php-sessions/) implements PHP’s native session handlers, backed by the WordPress database. This allows plugins, themes, and custom code to safely use PHP `$_SESSION`s in a distributed environment where PHP’s default tempfile storage just won’t work.
+
+This plugin is recommended for use on any site where a plugin or theme uses PHP Sessions.
+
+## Object Cache Pro
+
+<ReviewDate date="2023-05-23" />
+
+[Object Cache Pro](https://objectcache.pro/) remembers, or caches, any queries to the server after a Drupal or WordPress page is loaded for the first time. When another user loads the page, the results are provided from the Object Cache which is stored in memory without querying the database again. This results in much faster page load times, and less server impact on database resources.
+
+This plugin is recommended for use on any site using Redis and object caching.
+
+## Bulk Indexer for Yoast
```

Comment:
I'm unsure if we formally want to recommend this plugin at this time. I think ideally this would live in the [plugin repo](https://wordpress.org/plugins).